packages/eslint-plugin/base.config.js

```diff
@@ -0,0 +1,55 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
```

Comment:
The custom rules in this PR are quite extensive :clap:. Is the ecmaVersion used for linting being determined by this parser? What version is being used?

Reply:
`extends: ['plugin:@typescript-eslint/recommended']` pulls in this file https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/base.json which enables ES6 modules.  I don't think the ecmaVersion is explicitly set anywhere 🤔